SpectraLogic.EscapePodClient/ResponseParsers/DeleteClusterResponseParser.cs

```diff
@@ -0,0 +1,37 @@
+﻿/*
+ * ******************************************************************************
+ *   Copyright 2014-2018 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+using SpectraLogic.EscapePodClient.Runtime;
+using SpectraLogic.EscapePodClient.Utils;
+using System.Net;
+
+namespace SpectraLogic.EscapePodClient.ResponseParsers
+{
+    internal class DeleteClusterResponseParser : IResponseParser<bool>
+    {
+        #region Methods
+
+        public bool Parse(IHttpWebResponse response)
```

Comment:
This seems like it should return a Unit type, like Void or something.  Is that an option instead of returning `bool`?

Reply:
Not really since there is none in this version of .Net

Reply:
Is there no Void type?

Reply:
@rpmoore I've created a Void type that we can now use, let me know if you like it